client/package.json

```diff
@@ -20,8 +20,13 @@
     "localforage": "^1.10.0"
   },
   "dependencies": {
+    "@oneblink/apps": "^1.2.4",
     "@oneblink/sdk-core": "^0.4.1",
     "@oneblink/types": "github:oneblink/types",
-    "@types/lodash.clonedeep": "^4.5.7"
+    "@types/lodash.clonedeep": "^4.5.7",
+    "react": "^18.2.0"
```

Comment:
I would make this a dev dependency or a peer dependency.  We don't want to ship react with the package, but rather allow the user to use whatever version of react they are using. I think 18+ should be enough, you don't need to specify an exact version.